src/views/Home.vue

```diff
@@ -1,88 +1,97 @@
-                        </v-list-tile-action>
-                        <v-list-tile-content>
-                            <v-list-tile-title>
-                                {{ item.text }}
-                            </v-list-tile-title>
-                        </v-list-tile-content>
-                    </v-list-tile>
-                </template>
-            </v-list>
-        </v-navigation-drawer>
-        <v-toolbar class="primary" dark clipped-left>
-            <v-toolbar-side-icon @click.native="drawer = !drawer"></v-toolbar-side-icon>
+        <anti-pattern-tags-component v-model="selectedTags" :tags="tags"/>
+        <v-toolbar class="primary" app dark clipped-left>
             <span class="title ml-3 mr-5">Service-Based Antipatterns</span>
             <v-text-field
                     solo-inverted
                     flat
+                    clearable
+                    hide-details
                     label="Search"
-                    prepend-icon="search"
+                    prepend-inner-icon="search"
+                    v-model="searchTerm"
+                    @keydown.esc="clearSearch()"
             ></v-text-field>
             <v-spacer></v-spacer>
         </v-toolbar>
         <v-content>
-            <anti-patterns/>
+            <anti-patterns-container-component :anti-patterns="antiPatterns"/>
         </v-content>
     </div>
 </template>
 
 <script lang="ts">
-    import {Component, Vue} from 'vue-property-decorator';
-    import AntiPatterns from '@/components/AntiPatternsContainerComponent';
+    import {Component, Vue, Watch} from 'vue-property-decorator';
+    import axios from 'axios';
+    import AntiPatternsContainerComponent from '../components/AntiPatternsContainerComponent';
+    import AntiPatternTagsComponent from "../components/AntiPatternTagsComponent";
+    import {AntiPattern} from "../common/anti-pattern";
+    import Utils from "../utils/Utils";
 
     @Component({
-        components: {AntiPatterns},
+        components: {AntiPatternTagsComponent, AntiPatternsContainerComponent},
     })
     export default class Home extends Vue {
-        public data() {
-            return {
-                drawer: null,
-                items: [
-                    {icon: 'lightbulb_outline', text: 'Tags'},
-                    {divider: true},
-                    {icon: 'device_hub', text: 'Github'},
-                    {icon: 'help', text: 'Help'},
-                ],
-            };
+        private antiPatterns: AntiPattern[] = [];
+        private antiPatternsAll: AntiPattern[] = [];
+        private antiPatternsSelected: AntiPattern[] = [];
+        private antiPatternsFiltered: AntiPattern[] = [];
+        private selectedTags: string[] = [];
+        private searchTerm: string = "";
+
+        public created() {
+            axios.get(`/service-based-antipatterns/assets/result.json`).then((response) => {
+                this.antiPatternsAll = response.data.antiPatterns;
```

Comment:
Wieso rufst du hier gleich die filter funktion auf ?

Reply:
Sorry typo I mean `why do you **NOT** call the filter function in the line before